kafka.cpp

```diff
@@ -380,15 +383,26 @@ Kafka::send(const vector<Reading *> readings)
 		payload << "{ \"asset\" : " << quote(assetName) << ", ";
 		payload << "\"timestamp\" : " << quote((*it)->getAssetDateUserTime(Reading::FMT_ISO8601MS, true)) << ", ";
 		vector<Datapoint *> datapoints = (*it)->getReadingData();
+		bool isImage = true;
```

Comment:
Should this initial value be false, and you change that to true, if you see a image/DataBuffer DPV? Would current logic work fine if there is no image/DataBuffer DPV in the reading?

Reply:
Changed the name of  boolean variable as discussed.